app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughTopicListFragmentTest.kt

```diff
@@ -103,6 +105,48 @@ class WalkthroughTopicListFragmentTest {
     )
   }
 
+  // Checks if progressBar is gets hidden on scrolling to the second row of items
+  @Test
+  fun testWalkthroughWelcomeFragment_portrait_recyclerViewIndex4__progressBarIsHidden() {
```

Comment:
`testWalkthroughWelcomeFragment_recyclerViewIndex4_progressBarIsHidden`

Reply:
We might need more test like, the progress bar is visible in both configurations. Same with header text. 
Also, We need to test like progress bar visible then scroll it is not visible, so it will test the whole thing together. Also, it's not more like e2e test so it will work correctly on robolectric as well.

Reply:
Done, we can discuss adding more tests

Reply:
> We might need more test like, the progress bar is visible in both configurations. Same with header text.
> Also, We need to test like progress bar visible then scroll it is not visible, so it will test the whole thing together. Also, it's not more like e2e test so it will work correctly on robolectric as well.

I think we intend to do this only for landscape configuration because there's less space for recyclerview items only in landscape mode, so we might need to configure our code to work only in landscape orientation.
@rt4914 Can you please confirm?

Reply:
@prayutsu Yes, this should be done only for landscape-mobile.